lib/ios-crash-log.js

```diff
@@ -0,0 +1,63 @@
+import path from 'path';
+
+import _ from 'lodash';
+import _glob from 'glob';
+import { fs, getLogger } from 'appium-support';
+import B from 'bluebird';
+let glob = B.promisify(_glob);
+
+
+require('date-utils');
+let logger = getLogger('IOS-crash-log');
+
+class IOSCrashLog { 
+  constructor() {
+    this.prevLogs = [];
+    this.logsSinceLastRequest = [];
+    this.crashDir = path.resolve(process.env.HOME, 'Library', 'Logs', 'DiagnosticReports');
```

Comment:
personally, since this is just a global thing (unrelated to the state of the object), i'd make it a const up top, like:

```
const CRASH_DIR = ...;
```